spec/models/user_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { build(:user) }
+
+  # Test email validations
+  it { is_expected.to validate_presence_of(:email) }
+  it { is_expected.to validate_uniqueness_of(:email).case_insensitive }
+  it { should allow_value('user@example.com').for(:email) }
+  it { should allow_value('user123@example.com.uy').for(:email) }
+  it { should_not allow_value('userexample.com').for(:email) }
+  it { should_not allow_value('user123@example').for(:email) }
+
+  # Test password validations
+  it { is_expected.to validate_presence_of(:password_digest)}
+  it { should validate_length_of(:password).is_at_least(5).allow_blank }
+end
```

Comment:
In the future this file could have tons of different tests. We normally separate the test with `describe 'what you want to test'` to make the file more readable. In this case you could use `describe 'validations'`. 
Also the idea is to code simple enough so you don't need to upload comments explaining the code (some times it's necessary tho). I'd remove the comments